package.json

```diff
@@ -121,11 +120,6 @@
     "select",
     "ui"
   ],
-  "bundlesize": [
-    {
-      "path": "./dist/react-select.js"
```

Comment:
This doesn't really make sense now since we don't have a UMD build, there's probably a tool to do a bundle size check that bundles dependencies and minifies but I wasn't sure if having a bundle size check is a thing we care about having exactly right now so I just removed it.